cmd/protoc-gen-cpp-tableau-loader/hub.go

```diff
@@ -348,12 +348,11 @@ std::shared_ptr<MessagerMap> Hub::InternalLoad(const std::filesystem::path& dir,
                                                std::shared_ptr<const LoadOptions> options /* = nullptr */) const {
   // intercept protobuf error logs
   auto old_handler = google::protobuf::SetLogHandler(util::ProtobufLogHandler);
-  auto opts = ParseLoadOptions(options);
   auto msger_map = NewMessagerMap();
   for (auto iter : *msger_map) {
     auto&& name = iter.first;
     ATOM_DEBUG("loading %s", name.c_str());
-    auto mopts = ParseMessagerOptions(opts, name);
```

Comment:
One line difference due to golang functional option programming pattern
, so golang code must call `ParseOptions` func:
<img width="1309" height="179" alt="WXWorkCapture_17555962276389" src="https://github.com/user-attachments/assets/e45a4330-cde9-4b83-a694-f35b5eeed33c" />